analyzers/tests/SonarAnalyzer.UnitTest/TestCases/MemberShadowsOuterStaticMember.CSharp11.cs

```diff
@@ -1,22 +1,180 @@
-﻿public interface IMath<T>
+﻿public interface OuterInterfaceWithStaticMember
```

Comment:
Educational: This now became a huge test case. I was wondering if we really need all the combinations from here. Or they are already covered in the base test cases...

Reply:
I trimmed it about 50-60%